wire-ios-sync-engine/Source/E2EI/CRL/CertificateRevocationListsChecker.swift

```diff
@@ -32,6 +32,7 @@ public class CertificateRevocationListsChecker: CertificateRevocationListsChecki
     private let crlAPI: CertificateRevocationListAPIProtocol
     private let mlsConversationsVerificationUpdater: MLSConversationVerificationStatusUpdating
     private let selfClientCertificateProvider: SelfClientCertificateProviderProtocol
+    private let featureRepository: FeatureRepositoryInterface
```

Comment:
It seems like you only need the e2ei feature config. Could you just inject that instead of the entire feature repository?

Reply:
yes, changed